src/site/content/en/blog/access-devices/index.md

```diff
@@ -0,0 +1,162 @@
+Engineering a Bluetooth Lightbulb] article from Uri Shaked for an example on how to do that.
+
+## 6. Does this device expose HID interfaces? {: #hid }
+
+Click the button below and see if you can spot a HID device that may be of
+interest to you. Note that security-sensitive HID devices (such as FIDO HID
+devices used for stronger authentication) are not accessible in Chrome.
+
+<div class="w-text--center">
+<button class="w-button w-button--secondary w-button--with-icon"
+data-icon="videogame_asset" onclick="navigator?.hid?.requestDevice({ filters: [] })
+|| alert('This browser does not support WebHID. Try Chrome on desktop.')">
+Request HID device
+</button>
+</div>
+
+If so, try [WebHID] to open a HID connection to this device. The next step is to
+understand HID reports and report descriptors through [collections]. This can be
+really hard without vendor documentation for this device. Reverse-engineering it
+is also a possibility with tools like [Wireshark].
+
+## 7. Does this device expose USB interfaces? {: #usb }
+
+Click the button below and see if you can spot a USB device that may be of
+interest to you.
+
+<div class="w-text--center">
+<button class="w-button w-button--secondary w-button--with-icon"
+data-icon="usb" onclick="navigator?.usb?.requestDevice({ filters: [] })
+|| alert('This browser does not support WebUSB. Try Chrome.')">
+Request USB device
+</button>
+</div>
+
+If so, try [WebUSB] to communicate with this device over the USB port. Without
+clear documentation for this device and what USB commands this device supports,
+it's really hard **but** still possible with lucky guessing. Watch [Exploring
+WebUSB and its exciting potential] talk from Suz Hinton for instance. With tools
+like [Wireshark], you can also reverse-engineer this device by capturing raw USB
+traffic to understand it better.
+
+## 8. Does this device expose serial ports? {: #serial }
+
+Click the button below and see if you can spot a serial port that may be of
+interest to you.
+
+<div class="w-text--center">
+<button class="w-button w-button--secondary w-button--with-icon"
+data-icon="developer_board" onclick="navigator?.serial?.requestPort({ filters: [] })
+|| alert('This browser does not support the Serial API. Try Chrome on desktop.')">
+Request serial port
+</button>
+</div>
+
+If so, try the [Serial API] to read from and write to this device. Without clear
+vendor documentation for this device though, you'll need some dedicated tools to
+help you inspect serial traffic and understand it.
+
+## Acknowledgements {: #helpful }
+
+Thank you to [Reilly Grant](https://github.com/reillyeon) for review of this article.
+
+<span>Photo by <a
+href="https://unsplash.com/@darya_tryfanava?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Darya
+Tryfanava</a> on <a
+href="https://unsplash.com/?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>
```

Comment:
```suggestion
Photo by [Darya Tryfanava](https://unsplash.com/@darya_tryfanava) on [Unsplash](https://unsplash.com/).
```

Add the deeplink to the photo, for whatever reason the Unsplash widget only links to the photographer.

Reply:
Done.